tensorboard/components_polymer3/tf_categorization_utils/categorizationUtils.ts

```diff
@@ -12,187 +12,190 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-namespace tf_categorization_utils {
-  /**
-   * Functions to extract categories of tags and/or run-tag combinations
-   * from a run-to-tag mapping. The resulting categories can be fed to a
-   * `tf-category-pane`, and their items can be `<dom-repeat>`ed in a
-   * Polymer component.
-   */
```

Comment:
Should we preserve this comment?

Reply:
Sure: done, and updated to make it actually correct.